core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -244,6 +249,25 @@ internal suspend fun processTransientSettingEvents(
             }
         }
 
+        if (BuildCompat.isAtLeastV()) {
```

Comment:
Since we're no longer using the preview SDK, this should be changed to:
`Build.VERSION.SDK_INT >= Build.VERSION_CODES.VANILLA_ICE_CREAM`